docs/source/datasources/usgs_ndvi.rst

```diff
@@ -0,0 +1,131 @@
+This module is designed to allow programmatic access to and analysis of NDVI data from the data explorer.
+
+Usage
+-----
+
+To use this class, you first need to create a country configuration
+for the country you would like to analyze. You also need to supply the area
+of coverage that the country data is contained in. You can Make sure FEWS NET covers the country
+of interest. You can find area of coverage for NDVI by searching for eMODIS NDVI C6 on
+the  `USGS search portal <https://earlywarning.usgs.gov/fews/search>`_.
+The valid values of the area names are listed above under Background.
+
+An example country config for Ethiopia is:
+
+.. code-block:: python
+
+    iso3: eth
+    usgs_ndvi:
+        region_name: east-africa
+
+For a number of countries this config has been implemented already.
+For others, you can create a custom country config.
+Once you have created the config you can load it:
+
+.. code-block:: python
+
+    from aatoolbox import create_country_config
+    country_config = create_country_config(iso3="eth")
+
+Next you need to instantiate the USGS NDVI class with the country config. For this
+example, we will use the class for accessing temporally smoothed NDVI values. Since
+the size of the data we will be working with is so large, we will only look at a
+sample of data from a few dekads, corresponding to the dates in January:
+
+.. code-block:: python
+
+    from aatoolbox import UsgsNdviSmoothed
+    ndvi_smooth = UsgsNdviSmoothed(
+        country_config=country_config,
+        start_date="2021-01-01",
+        end_date="2021-01-31"
+    )
+
+We will first need to download the NDVI data if you haven't used the class before.
+The code downloads all data between `start_date` and `end_date`. Be careful if not
+setting start and end date during instantiation, the method will automatically
+look for data between the first available data in the 19th dekad of 2002 and today.
+We simply have to call the method to download available data.
+
+.. code-block:: python
+
+    ndvi_smooth.download()
+
+Since each NDVI file is downloaded as a .tif, we can load them in individually as
+rasters. We can do this for a specific date. This time we will specify the date
+as a year and dekad tuple, although specify as a `datetime.date` or string as
+above is equally appropriate.
+
+.. code-block:: python
+
+    ndvi_2021_01 = ndvi_smooth.load_raster(date=(2021, 01))
+
+If time series analysis is desired on the NDVI data, the user can manually do this
+using loaded in data arrays. However, given the size of the individual raster files,
+this module provides the `process()` method that calculate statistics for
+a given area. For this, we need to provide a geodataframe and column to aggregate to.
+
+Let's load the second-level administrative boundaries for Ethiopia from our country
```

Comment:
Ha I am such a detail-nerd, but this states second-level and you code uses zero-level

Reply:
God yeah, sorry, changed it because even second level was taking time!